sympy/controlsys/tests/test_lti.py

```diff
@@ -0,0 +1,286 @@
+from __future__ import division
+from sympy import (symbols, Matrix, zeros, simplify, eye, exp,
+                   ones, cosh, sinh, Integral, expand, together
+                   )
+from sympy import Rational as R
+from sympy.controlsys.lti import StateSpaceModel, TransferFunctionModel
+from mpmath import e
+
+
+def test_StateSpaceModel_create():
+    a1, a2, a3, a4 = symbols('a1:5')
+    ssm = StateSpaceModel(Matrix([a1]), Matrix([a2]),
+                          Matrix([a3]), Matrix([a4]))
+    assert ssm.represent[0] == Matrix([a1])
+    assert ssm.represent[1] == Matrix([a2])
+    assert ssm.represent[2] == Matrix([a3])
+    assert ssm.represent[3] == Matrix([a4])
+
+    b1, b2 = symbols('b1:3')
+    c1, c2 = symbols('c1:3')
+    d = symbols('d')
+
+    A = Matrix([[a1, a2],
+                [a3, a4]])
+    B = Matrix([b1,
+                b2])
+    C = Matrix([[c1, c2]])
+    D = Matrix([d])
+    ssm = StateSpaceModel(A, B, C, D)
+    assert ssm.represent[0] == A
+    assert ssm.represent[1] == B
+    assert ssm.represent[2] == C
+    assert ssm.represent[3] == D
+
+    s = symbols('s')
+    G = Matrix([[(4 * s - 10) / (2 * s + 1), 3 / (s + 2)],
+                [1 / ((2 * s + 1) * (s + 2)), (s + 1) / (s + 2)**2]])
+    ssm = StateSpaceModel(TransferFunctionModel(G))
+    assert ssm.represent[0] == Matrix([[-9 / 2, 0, -6, 0, -2, 0],
+                                       [0, -9 / 2, 0, -6, 0, -2],
+                                       [1, 0, 0, 0, 0, 0],
+                                       [0, 1, 0, 0, 0, 0],
+                                       [0, 0, 1, 0, 0, 0],
+                                       [0, 0, 0, 1, 0, 0]])
+    assert ssm.represent[1] == Matrix([[1, 0],
+                                       [0, 1],
+                                       [0, 0],
+                                       [0, 0],
+                                       [0, 0],
+                                       [0, 0]])
+    assert ssm.represent[2] == Matrix([[-6, 3, -24, 15 / 2, -24, 3],
+                                       [0, 1, 1 / 2, 3 / 2, 1, 1 / 2]])
+    assert ssm.represent[3] == Matrix([[2, 0],
```

Comment:
I can't find the definition of `def represent`. I don't understand what this means or why it isn't a method.